src/wallet/wallet.h

```diff
@@ -1544,5 +1542,162 @@ class AddSpendingKeyToWallet : public boost::static_visitor<KeyAddResult>
     KeyAddResult operator()(const libzcash::InvalidEncoding& no) const;
 };
 
+struct LogTx
```

Comment:
Almost all the logic in this struct should be moved out of `wallet.h` (into `wallet.cpp` or elsewhere).

Reply:
https://github.com/zcash/zcash/pull/4423/commits/4c1171bc4ceb780b3cb67c2847cb9bfbe8c368a2